lib/src/chat_view.dart

```diff
@@ -117,6 +117,11 @@ class DashChat extends StatefulWidget {
   /// Should the messages be shown in reversed order.
   final bool inverted;
 
+  /// topOfMessagesWidget is the widget that will be placed above all messages
+  /// at the top of the scroll. This widget will appear even if there are no
+  /// messages yet.
+  final Widget topOfMessagesWidget;
+
```

Comment:
@lane-faison what do you think of calling this `chatHeaderBuilder` to follow his pattern with `chatFooterBuilder`?

then we could insert it in this view like right between these 2 lines
https://github.com/TangoGroup/dash_chat/blob/7ecbb53e4dbf52d1cc562ab89f140a3d78d00ca9/lib/src/chat_view.dart#L491-L492

similar to his approach here?
https://github.com/TangoGroup/dash_chat/blob/7ecbb53e4dbf52d1cc562ab89f140a3d78d00ca9/lib/src/chat_view.dart#L561-L562

Reply:
I like it, @lisamartin00. I will make this update. I am going to do `messageListHeaderBuilder` since the widget will live at the top of the `MessageListView`. `chatHeaderBuilder` would imply that the widget will go at the top of `ChatView`, but it needs to go inside of `MessageListView` and at the top.